cadc-storage-permissions/src/intTest/resources/testPermissionsClient/baldur.properties

```diff
@@ -0,0 +1,6 @@
+users = cn=cadcregtest1_b5d,ou=cadc,o=hia,c=ca
```

Comment:
cadcauthtest1 is normally the user that is allowed to do things
cadcregtest1 is a registered user with no special permissions

Is this file here for reference in setting up the local intTest? I don't see where it is used otherwise...

Reply:
A check of the X500 dn in the cert against the dn in the properties file gives access. The properties is just for the intTest, and the readme should be updated to explain it, which I'll do. Could also use a self signed cert instead of an internal cert.